### PR TITLE
NEWS/version for 2.53.3

### PR DESCRIPTION
**Pull request by gdt**

### CODE REVIEW

**Review by tleedjarv — COMMENTED** [review]
I'll not insist on adding these, but I can see two more bullets added:

Improved ETA calculation and sync speed display in text UI.

It could be worth mentioning that the copyquoterem preference has been removed, as this is a user-visible change.

File: NEWS.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ As of 2022, this file (NEWS.md at top level) is used for news.
 Some software has a "changelog" file that records all changes, but
 unison uses git history for that, and thus there is no changelog file.
 
+## Changes in 2.53.3
+
+Released 2023-04-28
+
+  * Enable syncing ACLs on Linux
```
[the remark this answers is not in the document]
Comment (tleedjarv) [reply NEWS.md:19]: Perhaps this wording is better. It's not that these attrs can't be synced to/from other platforms, it's that only Linux will interpret and enforce these as actual ACLs.
```suggestion
  * Enable syncing POSIX draft ACLs on Linux via xattrs
```

**Comment by gdt** [comment]
As I understand it the xattr impl of POSIX draft ACLs is nonstandard.   That's fine to support but I don't want to add to the already large confusion around ACLs.

**Comment by gdt** [comment]
I realize POSIX.1e was withdrawn, but it seems other systems implement it and Linux sort of does but that interfaces in 1003.2c aren't usable and thus a single code path doesn't work.   Thus "syncing POSIX draft ACLs" implies interop which isn't there.

**Pull request merged**